participant-goal/participant-goal.js

```diff
@@ -27,6 +31,14 @@
 		const $goal = $('#goal');
 		$raised.html(ELT.toCurrency(result.sumDonations));
 		$goal.html(ELT.toCurrency(result.fundraisingGoal));
+
+		if (ELT.settings.progressBar) {
+			const $progress = Math.min(100, (result.sumDonations/result.fundraisingGoal) * 100)
```

Comment:
I thought the JQuery progress bar you could set the max and value and it would do the division for you?